config/locales/en/providers.yml

```diff
@@ -62,16 +62,17 @@ en:
             the prospects of success
     check_benefits:
       index:
-        heading_1: Your client's tax and benefits status
-        heading_2: What happens next
+        next_heading: What happens next
+        central_paragraph: This is because they do not receive benefits that qualify for legal aid.
+        list_intro: 'You''ll need to:'
         negative_result:
-          body: Your client does not receive benefits that qualify for legal aid.
-          next: Your client will have to complete a financial means assessment. They can complete the financial assessment online from home or another location.
-          title: must complete a financial assessment
+          title: "%{name} must complete a financial assessment"
+          next_1: Your client will need to answer questions about their financial situation.
+          next_2: 'We''ll ask you to give them access to the online assessment which they can complete at home or at another location.'
         positive_result:
-          body: Your client receives benefits that qualify for legal aid.
-          next: Your client will have to declare whether they have any savings, own a home or anything else that can pay for legal aid.
-          title: receives a qualifying benefit
+          title: "%{name} receives benefits that qualify for legal aid"
+          next_1: 'answer questions about your client''s property, savings and other assets'
+          next_2: provide details of the case
```

Comment:
Use keys that describe the text content rather than generic names - it is then much clearer in the markup what is being passed in. So I'd suggest:
```yaml
what_happens_next: What happens next
because_not_receiving_benefits: This is because they do not receive benefits that qualify for
legal aid.
provider_needs_to: "You'll need to:"
negative_result:
  must_complete_assessment: "%{name} must complete a financial assessment"
  must_answer_financial_questions: Your client will need to answer questions about their financial situation.
  give_access_to_assessment: We'll ask you to give them access to the online assessment which they can complete at home or at another location.
positive_result:
  citizen_received_benefits: "%{name} receives benefits that qualify for legal aid"
  must_answer_saving_questions: answer questions about your client's property, savings and other assets
  must_provide_case_details: provide details of the case
```
With those keys someone looking at the markup, won't need to refer to the translation file all the time to get a gist of what is being displayed on the page.

Reply:
I'm not sure I should change `title` as this is used in the page template, and if I am using completely different keys for `positive_result` and `negative_result` then there's no need for that distinction.  

So I have changed the global keys as you suggested, and changed the keys with positive/negative differences to keys that make sense in both situations.